src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.vb

```diff
@@ -856,10 +859,7 @@ End Structure
                 references:={MscorlibRef, SystemRef, compilation0.EmitToImageReference(embedInteropTypes:=True)})
 
             Dim expectedOperationTree = <![CDATA[
-IInvalidExpression (OperationKind.InvalidExpression, Type: I, IsInvalid) (Syntax: 'New I(x)')
-  Children(2):
-      IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: System.Object, IsInvalid) (Syntax: 'x')
```

Comment:
It looks like we are losing information about parameter reference, this feels wrong, please open an issue to follow up.

Reply:
opened - https://github.com/dotnet/roslyn/issues/22229

...

the information is still there, just user gets the one of children initially that points to same syntax node rather than InvalidExpression. user can use Parent and children to get to the ParameterReferenceExpression.